TrafficCapture/trafficReplayer/src/test/resources/log4j2.properties

```diff
@@ -1,7 +1,7 @@
 status = error 
 
 # Root logger options
-rootLogger.level = debug
+rootLogger.level = info
```

Comment:
Will it be helpful if for a future task, debug level logger can be added to a debug mode? I don't recall having debug mode settable in a parameter.